apps/nextjs-website/src/app/auth/change-password/page.tsx

```diff
@@ -0,0 +1,81 @@
+'use client';
+import { Alert, Box, Grid, Snackbar } from '@mui/material';
+import PasswordChangedCard from '@/components/organisms/Auth/PasswordChangedCard';
+import ChangePasswordForm from '@/components/organisms/Auth/ChangePasswordForm';
+import { ResetPasswordSteps } from '@/lib/types/resetPasswordSteps';
+import { useCallback, useState } from 'react';
+import { useSearchParams } from 'next/navigation';
+import { Auth } from 'aws-amplify';
+
+const ChangePassword = () => {
+  const searchParams = useSearchParams();
+  const username = searchParams.get('username') || '';
+  const code = searchParams.get('code') || '';
+
+  const [resetPasswordSteps, setResetPasswordSteps] = useState(
+    ResetPasswordSteps.CHANGE_PASSWORD
+  );
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState<string | null>(null);
+
+  const OnChangePassword = useCallback(async () => {
+    const success = await Auth.forgotPasswordSubmit(
+      username.replace(' ', '+'),
```

Comment:
```suggestion
      username,
```

Reply:
This is a little workaround for the email aliases, since for testing we are using them and we have to write them in the query string and are not parsed correctly

Reply:
The system will send the `sub` parameter, not the `email`. Moreover, the system should work also with aliases. IMHO let's try without this replace